canonical-web-and-design/ubuntu.com
# Update the advantage dashboard to display ESM apps contracts

## Done
- Add ESM apps to be displayed in contracts dashboard on the advantage page
- Linted the styles.scss file

**Notes for UX**
There is nothing clear in the response that indicates weither the contract is exclusively an apps or infra contract other than the esm affordance. I wonder if it's clear enough with the "Apps" and "Infra" text in the esm cell?

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/advantage
- If you have contracts check it matches [the wireframe](https://user-images.githubusercontent.com/19801137/76755183-53a23880-677b-11ea-9053-b1d33971c295.png) if not check the screenshots

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2353

## Screenshots
![Screenshot_2020-03-19 Ubuntu Advantage for Infrastructure Ubuntu](https://user-images.githubusercontent.com/1413534/77123909-56ca4c80-6a39-11ea-843e-e5d44cbba683.png)

![Screenshot_2020-03-19 Ubuntu Advantage for Infrastructure Ubuntu(1)](https://user-images.githubusercontent.com/1413534/77123914-5a5dd380-6a39-11ea-8b87-d4f1d83ab203.png)

![Screenshot_2020-03-19 Ubuntu Advantage for Infrastructure Ubuntu(2)](https://user-images.githubusercontent.com/1413534/77123919-5d58c400-6a39-11ea-96c1-4b983ad9e336.png)
